scripts/ci

```diff
@@ -1,2 +1,14 @@
 #!/usr/bin/env bash
-./node_modules/.bin/karma start karma.devicelab.js --single-run
+
+
+# List of browsers to test
+BROWSERS=(
+    bs_chrome_50
+    bs_firefox_47
+    bs_safari_8
+    bs_ie_11
+)
+
+for B in ${BROWSERS[@]}; do
+    BROWSER=$B ./node_modules/.bin/karma start karma.devicelab.js --single-run
+done
```

Comment:
Let's leave it for now, and we'll update it when it goes into OMNI (slightly different layout)